styles.css

```diff
@@ -163,4 +163,4 @@ form > button {
 form > button:hover {
   background: #e07999;
   color: white;
-}
\ No newline at end of file
+}
```

Comment:
It looks to me like nothing actually changed in this file. I'm not sure why git ends up tracking stuff like this, I think it's an accident. It's nice to keep an eye out for "changes" like this when you're preparing to commit and unstage them so that they don't end up in our commits, keeps things tidier and easier to understand (since we didn't actually intend to change anything in styles.css, it could be confusing to see git saying "changed styles.css")